src/data.js

```diff
@@ -2064,6 +2064,26 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Tony Mamedbekov',
+    description:
+      'Architect, developer, cms enthusiast, technologoy evangelist, content creator, bloger, consultant',
+    url: 'https://www.tmamedbekov.dev',
```

Comment:
Please update the url to `https://tmamedbekov.dev/uses/` and commit again. Thanks!